tests/testthat/test-tidy-upkeep.R

```diff
@@ -12,6 +12,41 @@ test_that("upkeep bullets don't change accidentally", {
   create_local_package()
 
   expect_snapshot(writeLines(
-    upkeep_checklist(posit_pkg = TRUE, posit_person_ok = FALSE)
+    tidy_upkeep_checklist(posit_pkg = TRUE, posit_person_ok = FALSE)
   ))
 })
+
+test_that("upkeep bullets don't change accidentally",{
+  skip_if_no_git_user()
+  withr::local_options(usethis.description = NULL)
+  create_local_package()
+
+  # Need to add git so can detect presence of master branch
```

Comment:
I would use the new `testthat::local_mocked_bindings()` to eliminate the fussiness. I.e. just fake having a git repo with a `master` branch.

https://testthat.r-lib.org/reference/local_mocked_bindings.html

Reply:
Oh great, I haven't used this yet!

Reply:
That's really nice - thanks for the pointer!